src/components/jar_details/JarDetailsOverlay.module.css

```
@@ -0,0 +1,236 @@
+  min-height: 3.6rem;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 0.5rem;
+  padding: 0 0 0.8rem 0;
+  background-color: var(--bs-gray-100);
+}
+
+@media only screen and (min-width: 768px) {
+  .overlayBody .tabContainer > .utxoListTitleBar {
+    align-items: center;
+    padding: 0.8rem 1rem;
+    border-radius: 0.6rem;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .overlayBody .tabContainer > .utxoListTitleBar {
+    flex-direction: row;
+  }
+}
+
+:root[data-theme='dark'] .overlayBody .tabContainer > .utxoListTitleBar {
+  background-color: var(--bs-gray-800);
+}
+
+.overlayBody .tabContainer > .utxoListTitleBar .freezeUnfreezeButtonsContainer {
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+}
+
+.overlayBody .tabContainer > .utxoListTitleBar .freezeUnfreezeButtonsContainer > button {
+  width: 8rem;
+  font-size: 0.6rem;
+  padding: 0.3rem 0.5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.3rem;
+  background-color: white;
+  border: 1px solid var(--bs-gray-200);
+  color: var(--bs-body-color);
+  border-radius: 0.35rem;
+}
+
+.overlayBody .tabContainer > .utxoListTitleBar .freezeUnfreezeButtonsContainer > button:hover {
+  background-color: var(--bs-btn-hover-bg);
+  border-color: var(--bs-btn-hover-border-color);
+}
+
+@media only screen and (min-width: 768px) {
+  .overlayBody .tabContainer > .utxoListTitleBar .freezeUnfreezeButtonsContainer > button {
+    width: 10rem;
+    font-size: 0.8rem;
+  }
+}
+
+:root[data-theme='dark'] .overlayBody .tabContainer > .utxoListTitleBar .freezeUnfreezeButtonsContainer > button {
+  background-color: var(--bs-gray-600) !important;
+  border-color: var(--bs-gray-600);
+}
+
+:root[data-theme='dark'] .overlayBody .tabContainer > .utxoListTitleBar .freezeUnfreezeButtonsContainer > button:hover {
+  background-color: var(--bs-gray-700) !important;
+  border-color: var(--bs-gray-700);
+}
+
+.overlayBody .tabContainer > .utxoListTitleBar .refreshButton {
+  width: 2rem;
+  padding: 0.1rem;
+  border: none;
+}
+
+.overlayBody .accountItem {
+  color: var(--bs-body-color);
+  margin-bottom: 1rem;
+  box-shadow: 0px 1px 2px var(--bs-gray-400);
+  border: none;
+  border-radius: 0.5rem !important;
+  background-color: var(--bs-gray-100);
+}
+.overlayBody .accountItem:last-child {
+  margin-bottom: 0;
+}
+
+:root[data-theme='dark']
+  .overlayBody
+  .accountItem
+  :global
+  .card:not(.border-success):not(.border-danger):not(.border-warning):not(.border-primary):not(.border-info) {
+  border-color: var(--bs-gray-700) !important;
+}
+
+:root[data-theme='dark'] .overlayBody .accountItem {
+  background-color: var(--bs-gray-800) !important;
+  box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.5);
+}
+
+.overlayBody .accountItem > h2 > button {
+  border-radius: 0.5rem !important;
+  background-color: var(--bs-gray-100);
+  box-shadow: none;
+}
```

```suggestion
.overlayBody .accountItem > h2 > button {
  border-radius: 0.5rem !important;
  background-color: var(--bs-gray-100);
  box-shadow: none;
}

:root[data-theme='dark'] .overlayBody .accountItem > h2 > button {
  background-color: transparent;
}
```

Fixes background color of accordion header in dark-mode.

---

Maybe we should strive to have more general style rules and move these to `index.css`.

e.g. something like this in `index.css`:
```css
.accordion {
  --bs-accordion-bg: var(--bs-gray-100);
}

.accordion-button {
  background-color: transparent;
}
```